server/storage/storage_impl.go

```diff
@@ -85,129 +83,129 @@ func (s *metaStorageImpl) CreateCluster(ctx context.Context, cluster *clusterpb.
 		Then(opCreateCluster).
 		Commit()
 	if err != nil {
-		return nil, errors.WithMessagef(err, "fail to create cluster, clusterID:%d, key:%s", cluster.Id, key)
+		return errors.WithMessagef(err, "create cluster, clusterID:%d, key:%s", req.Cluster.ID, key)
 	}
 	if !resp.Succeeded {
-		return nil, ErrCreateClusterAgain.WithCausef("cluster may already exist, clusterID:%d, key:%s, resp:%v", cluster.Id, key, resp)
+		return ErrCreateClusterAgain.WithCausef("cluster may already exist, clusterID:%d, key:%s, resp:%v", req.Cluster.ID, key, resp)
 	}
-	return cluster, nil
+	return nil
 }
 
-// Return error if the cluster topology already exists.
-func (s *metaStorageImpl) CreateClusterTopology(ctx context.Context, clusterTopology *clusterpb.ClusterTopology) (*clusterpb.ClusterTopology, error) {
-	now := time.Now()
-	clusterTopology.CreatedAt = uint64(now.Unix())
-
-	value, err := proto.Marshal(clusterTopology)
+// CreateClusterView return error if the cluster view already exists.
+func (s *metaStorageImpl) CreateClusterView(ctx context.Context, req CreateClusterViewRequest) error {
+	pb := ConvertClusterViewToPB(req.ClusterView)
```

Comment:
I think this variable name is fuzzy, maybe rename it as `ClusterTopology` will better.

Reply:
I rename it to `clusterViewPB`.